### PR TITLE
Add sample RUM session info for all frameworks

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

Setting the sampling rate for RUM sessions is very important to customers and should not be hidden within advanced configuration instructions. The instructions have been moved or added to all setup pages as per [DOCS-7658](https://datadoghq.atlassian.net/browse/DOCS-7658).

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

Should be reviewed by @MaelNamNam or at least one person on the rum-mobile team before merging.

[DOCS-7658]: https://datadoghq.atlassian.net/browse/DOCS-7658?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brett0000FF — APPROVED** [review]

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md [hunk 1]
```diff
@@ -228,3 +232,4 @@ Whenever you perform an operation that might throw an exception, you can forward
 [7]: /getting_started/tagging/using_tags/#rum--session-replay
 [8]: /real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/roku#track-rum-resources
 [9]: /real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/roku#enrich-user-sessions
+[10]:
```
Comment (fuzzybinary) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md:235]: Extra link anchor?

**Review by fuzzybinary — APPROVED** [review]
Looks good.  Does Flutter already have this information in the right place?

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md:235]: Yes, thanks for flagging! Removing.

**Comment by rtrieu** [comment]
@fuzzybinary Yes it does, thanks for checking! :)

**Pull request merged**